deployment/dev/Dockerfile

```diff
@@ -1,3 +1,27 @@
-FROM ubuntu:latest
+FROM python:3.12-slim
 
-ENTRYPOINT ["top", "-b"]
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1 \
+    PYTHONUNBUFFERED 1
+
+RUN apt-get update \
+    && apt-get -y install curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install poetry
+
+COPY ./pyproject.toml ./poetry.lock /app/
```

Comment:
shouldn't we use the less number of COPY commands?

Reply:
Fixed.